package.json

```diff
@@ -8,7 +8,7 @@
     "build:static": "rimraf .svelte build/static && svelte-kit build",
     "check": "svelte-check --ignore build,.svelte",
     "check:dev": "svelte-check --ignore build,.svelte --watch",
-    "dev": "run-p -r dev:serve check:dev lint:dev",
+    "dev": "run-p -r graphql-codegen:prod:watch dev:serve check:dev lint:dev",
```

Comment:
this is needed for the typescript and svelte. i think the artifact is on your local so you haven't seen this issue. these scripts aren't run in order, but the codegen step writes a file to disk that triggers a rebuild and stops it from being broken.